### ISSUE TITLE
Plugin fail to work with Jmeter's Keystore Configuration

### ISSUE THREAD

**Issue by syampol13** [Blazemeter/jmeter-http2-plugin#49]
I have some endpoints where I need to use client certificate. I'm using [Keystore_Configuration](https://jmeter.apache.org/usermanual/component_reference.html#Keystore_Configuration) for that.
Certificate is valid and does work with the default HTTP sample.
But when using with the HTTP2 sample - I'm getting following error:
![image](https://github.com/Blazemeter/jmeter-http2-plugin/assets/114644227/beed275a-2b3f-4039-8cbc-615f14dcc229)
Similar for the *nix host (so it's not a path issue)

What is even more weird is that I got same error even if the target for the endpoint used within the HTTP2 sample doesn't require any client cert.

Can't share original keystore with the client cert. But created a dummy TP (can't guarantee the attached cert is fully valid. but if it's not - you can use your own). [TP + cert + jmeter.log](https://github.com/Blazemeter/jmeter-http2-plugin/files/11575381/assets.zip)
TP perform two simple GET. One via HTTP1.1 and another via HTTP2. None of those required client certificate and both executed successfully.

But if I add following to the system.properties:
```
javax.net.ssl.keyStore=PATH_TO_ATTACHED_CERT/clnt_certs_test.p12
javax.net.ssl.keyStorePassword=testks
```
Then HTTP2 sample will fail with the above error.

Keystore cfg:
![image](https://github.com/Blazemeter/jmeter-http2-plugin/assets/114644227/d0a2c744-eb01-43cf-93db-c75e1fba6470)

**Comment by syampol13** [comment]
seems that there is still a problem with the path on WIN. it must start with /

**Comment by 3dgiordano** [comment]
Hi @syampol13 

Version 2.0.2 has bugs and does not properly support a Keystore file

We recommend that you try version 2.0.3 where we supported it.
That version is still in alpha and you can download it from

https://github.com/Blazemeter/jmeter-http2-plugin/releases/tag/v2.0.3-alpha.3

Let us know if that version works for you.

**Comment by 3dgiordano** [comment]
Hi @syampol13

Final release of version 2.0.3 was done today.
https://github.com/Blazemeter/jmeter-http2-plugin/releases/tag/v2.0.3

Soon it will be available in JMeter Plugins Manager.

**Comment by 3dgiordano** [comment]
Hi @syampol13

The issue was resolved on the 2.0.3 release.
This issue is closed due to inactivity.
If the issue is not resolved in that version, you can reopen the issue or report it in a new one.
